src/pages/Verify/index.tsx

```diff
@@ -40,41 +41,68 @@ enum STATUS {
   ERROR = 'Error occur, please try again.',
 }
 
+function validateUrl(url: string) {
+  if (!url) return ''
+  const whitelistDomain = ['https://blog.kyberswap.com']
+  try {
+    if (whitelistDomain.some(e => new URL(e).host === new URL(url).host)) return url
```

Comment:
This is unsafe, `new URL('asdasd')` will throw an error

Reply:
added try catch

Reply:
Ah ok, I see the try catch. But why not just comparing with `startsWith`?

Reply:
for sure , it will work for both
http://blog.kyberswap.com
https://blog.kyberswap.com